exporter/otlphttpexporter/otlp.go

```diff
@@ -147,6 +147,8 @@ func (e *baseExporter) export(ctx context.Context, url string, request []byte) e
 		// Request is successful.
 		return nil
 	}
+	data, _ := io.ReadAll(resp.Body)
+	fmt.Println("RESPONSE", resp, string(data))
```

Comment:
Is it intentional to keep this console output?

Reply:
(No, this is the thing I've been debugging. Sorry!)